ocaml/sdk-gen/csharp/FriendlyErrorNames.resx

```diff
@@ -114,7 +114,7 @@
   <data name="HOST_CANNOT_ATTACH_NETWORK" xml:space="preserve">
     <value>Server cannot attach network (in the case of NIC bonding, this may be because attaching the network on this server would require other networks [that are currently active] to be taken down).</value>
   </data>
-  <data name="HOST_CANNOT_ATTACH_NETWORK-SHORT" xml:space="preserve">
+  <data name="HOST_CANNOT_ATTACH_NETWORK_SHORT" xml:space="preserve">

```

Comment:
This needs a change in Failure.cs (personally I'd like to remove those because they are xencenterisms and they rather belong there and not in the SDK, but it can wait for a future PR).